src/components/friends/FriendCard.tsx

```diff
@@ -64,7 +64,7 @@ const FriendCard: React.FC<Friend> = (friend) => {
       <Content>{friend.description}</Content>
       <ButtonSection>
         {friend.slug &&
-          <Link to={friendUrl(friend.slug, hash.friends.friend.id)}>
+          <Link to={buildFriendUrl(friend.slug, hash.friends.friend.id)}>
```

Comment:
generalnie cały koncept z urlami jest taki, żeby był jeden obiekt urls, który ma w sobie właśnie funkcje tworzące urle - buildery

Reply:
ok, teraz chyba rozumiem, czego nie wyjaśniłem - a nie miałem czasu się skupić w ogóle na naszym projekcie

Reply:
chodziło mi o to, że w "urls" możemy trzymać nie tylko statyczne wartości (czyli konkretne URLe), ale mogą to być także funkcje

Reply:
czyli może być np. urls.pola.friends(friend.id)
funkcja a nie stały url
w powyższym przypadku, bez friend.id ta funkcja zwracałaby po prostu adres podstrony przyjaciół

Reply:
zatem wywołanie urls.pola.friends('nowy-przyjaciel') mogłoby zwracać:
/friends#nowy-przyjaciel